src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs

```diff
@@ -220,6 +232,15 @@ private IList<ManifestItem> FindTocInManifest(Manifest manifest)
             return manifest.Files.Where(f => IsType(f, ManifestItemType.Toc)).ToList();
         }
 
+        private ManifestItem FindSiblingCoverPageInManifest(Manifest manifest, ManifestItem tocFile)
+        {
+            return manifest.Files.SingleOrDefault(f =>
+            {
+                return Path.GetDirectoryName(f.SourceRelativePath).StartsWith(Path.GetDirectoryName(tocFile.SourceRelativePath))
+                    && (string.Compare("cover.md", Path.GetFileName(f.SourceRelativePath), true) == 0);
```

Comment:
use `FilePathComparer.OSPlatformSensitiveRelativePathComparer.Equals(..., ...)` This should be case sensitive under Linux.

Reply:
FYI, Windows 10 can be configured for case sensitivity by folder.
https://www.windowscentral.com/how-enable-ntfs-treat-folders-case-sensitive-windows-10

`FilePathComparer.OSPlatformSensitiveRelativePathComparer` will replace back slashes with forward slashes.